Sources/StreamVideo/HTTPClient/InternetConnection.swift

```diff
@@ -168,7 +169,7 @@ extension InternetConnection {
             guard monitor == nil else { return }
             
             monitor = createMonitor()
-            monitor?.start(queue: .global())
+            monitor?.start(queue: queue)
```

Comment:
Would it make more sense to use a global queue with a qos of `.background` or `.utility`?

Reply:
I would prefer to keep it like this, since this was already tested on the chat monitor on older iOS 14/15 versions.